### ISSUE TITLE
Swiping offset not working on iOS

### ISSUE THREAD

**Issue by nandorojo** [breeffy/react-native-calendar#37]
https://user-images.githubusercontent.com/13172299/114905059-87b8b480-9de6-11eb-9b25-d0e850570f65.mp4

**Comment by likern** [comment]
@nandorojo Can this be related to  **Gilroy-Medium** font?
Could you try to use default font (by providing custom theme object)?

As I understand on Android custom font is substituted to default font. So I have no issues in example application (even though I don't have **Gilroy-*** fonts there).

**Comment by nandorojo** [comment]
Where should I edit the font? The theme is a pretty deep object.

**Comment by likern** [comment]
@nandorojo Yes. You have to substitute all objects with `textFont` property like here https://github.com/breeffy/react-native-calendar/blob/6db66e147d514c5a9b97bb0588c3db2ff7b16b4b/src/themes/CalendarThemeLight.ts#L17

You can use `CalendarThemeLight` as base object and change (using destructuring) properties where font is used.
Or, for simplicity, you can just directly modify this file to test out things.
